Sources/Classes/core/API/PreviewMiniappAPI.swift

```diff
@@ -15,9 +15,10 @@ internal class PreviewMiniappAPI {
     }
 
     private func getMiniAppInfo(using token: String) -> URL? {
+        let previewToken = token.replacingOccurrences(of: "\\\\/", with: "", options: .regularExpression, range: nil)
```

Comment:
Do you really need a regex for that, seems over-engineering to me ^^

Reply:
Yeah, I was thinking of an easy approach but sometimes what if we get too many / in the preview code? in that case regex would be easy solution